controllers/keystoneapi_controller.go

```diff
@@ -202,8 +195,8 @@ func (r *KeystoneAPIReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *KeystoneAPIReconciler) reconcileDelete(ctx context.Context, instance *keystonev1.KeystoneAPI, helper *helper.Helper) (ctrl.Result, error) {
-	r.Log.Info("Reconciling Service delete")
-
+	logFromCtx := log.FromContext(ctx)
```

Comment:
(here and below): I would shorten the name just to `l := log.FromContext(ctx)` to save some typing. :)